doc/specs/vulkan/chapters/fundamentals.txt

```diff
@@ -600,6 +602,16 @@ library) perform memory barriers as a part of mutual exclusion, so mutexing
 Vulkan objects via these primitives will have the desired effect.
 ====
 
+Similarly the application must: avoid any potential data hazard of
+application-owned memory that has its
+<<fundamentals-objectmodel-lifetime-acquire,ownership temporarily acquired>>
+by a Vulkan command.
+While the ownership of an application-owned memory remains acquired by a
```

Comment:
Here again, is "an application-owned memory" intended/correct? Why the "an"?

Reply:
Because I have no idea how English works anymore. :) Care to write a specification about that? :p 
It is singular noun so I just assume there should be "an" or "the" or "this" or "any" or some such.

Reply:
Memory is a mass noun, rather than a singular, so it doesn't get an article (a/an/the). If you said "memory object", that would be singular. and thus get an article.

Reply:
I guess...